### PR TITLE
Renamed unit tests to Uniform naming

### PR DESCRIPTION
**Pull request by entvex**
Hello dear RiotSharp community.

I have renamed the unit tests. 

I would also suggest changing some of them so there is only one assert statement and maybe switching to NUnit or xUnit, but these things can be discussed in another issue 😊

### CODE REVIEW

File: RiotSharp.Test/CacheTest.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ public class CacheTest
 
         [TestMethod]
         [TestCategory("Cache")]
-        public void AddGet_TimeSpan_ShouldAddToTheCache_Test()
+        public void AddGet_TimeSpan_ShouldAddToTheCache_ReturnTrue()
```
Comment (BenFradet) [inline RiotSharp.Test/CacheTest.cs:17]:
```suggestion
        public void AddGet_TimeSpan_ShouldAddToTheCache_ReturnsTrue()
```

File: RiotSharp.Test/CacheTest.cs [hunk 2]
```diff
@@ -46,7 +46,7 @@ public void AddGet_DateTime_ShouldAdd_Test()
 
         [TestMethod]
         [TestCategory("Cache")]
-        public void AddGet_DateTime_ShouldAddAndExpire_Test()
+        public void AddGet_DateTime_ShouldAddAndExpire_ReturnTrue()
```
Comment (BenFradet) [inline RiotSharp.Test/CacheTest.cs:49]:
```suggestion
        public void AddGet_DateTime_ShouldAddAndExpire_ReturnsTrue()
```

**Review by BenFradet — COMMENTED** [review]
I'd suggest standardizing between `Return` and `Returns`

File: RiotSharp.Test/CacheTest.cs (hunk 1 above)
Comment (entvex) [reply RiotSharp.Test/CacheTest.cs:17]: Do you think we should change the name to AddGet_TimeSpanShouldAddToTheCache_ReturnsTrue

So we keep the MethodName_StateUnderTest_ExpectedBehavior() naming convention?

Thanks for reviewing my pull request 👍

Comment (BenFradet) [reply RiotSharp.Test/CacheTest.cs:17]: yes, I think the more standardization the better :+1:

**Comment by entvex** [comment]
As you said in an earlier comment returns have been changed to return in the tests 👍

File: RiotSharp.Test/CacheTest.cs [hunk 3]
```diff
@@ -154,15 +156,15 @@ public void Keys_ShouldGetAllKeys_Test()
 
         [TestMethod]
         [TestCategory("Cache")]
-        public void Keys_ShouldGetEmptyIfEmptyCache_Test()
+        public void Keys_ShouldGetEmptyIfEmptyCache_ReturnsTrue()
```
Comment (BenFradet) [inline RiotSharp.Test/CacheTest.cs]:
```suggestion
        public void Keys_ShouldGetEmptyIfEmptyCache_ReturnTrue()
```

File: RiotSharp.Test/RiotApiTest.cs [hunk 4]
```diff
@@ -92,35 +92,7 @@ public void GetChampionRotationAsync_Test()
...
-        {
-            EnsureCredibility(() =>
-            {
-                // TODO: Properly implement League id test
-                return;
-                var leagues = Api.League.GetLeagueByIdAsync(RiotApiTestBase.SummonersRegion, "LEAGUE-ID-HERE");
-
-                Assert.IsTrue(leagues.Result.Queue != null);
-            });
-        }
```
Comment (BenFradet) [inline RiotSharp.Test/RiotApiTest.cs]: why was this removed?

**Review by BenFradet — COMMENTED** [review]
Looking good :+1:

File: RiotSharp.Test/RiotApiTest.cs (hunk 4 above)
Comment (entvex) [reply RiotSharp.Test/RiotApiTest.cs]: I removed two tests, because they both had TODO's with "Properly implement"

So i just assumed they where not in use or will be fixed because the tests with TODO's seem very old.

Please let me know if you would like them added back in :)

Comment (BenFradet) [reply RiotSharp.Test/RiotApiTest.cs]: let's leave them in :+1:

Comment (entvex) [reply RiotSharp.Test/RiotApiTest.cs]: I have added them back in and renamed them 👍

**Review by BenFradet — APPROVED** [review]
